### PR TITLE
Fixed order model

### PR DESCRIPTION
**Pull request by kapoliub**
- removed populate methods from service
- fixed calculation total items price
- added populating to resolver.js for orders

### CODE REVIEW

**Review by jul4ajka — APPROVED** [review]

File: modules/order/order.resolver.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const ordersMutation = {
     try {
       return await ordersService.addOrder(args.order);
     } catch (e) {
+      console.log(e);
```
Comment (moran711) [inline modules/order/order.resolver.js:29]: Please remove this `console.log`.

File: resolvers.js [hunk 2]
```diff
@@ -233,6 +279,7 @@ const resolvers = {
       materialsService.getMaterialById(parent.bottomMaterial),
     bottomColor: parent => colorService.getColorById(parent.bottomColor),
   },
+  ConstructorBottom: {},
```
Comment (moran711) [inline resolvers.js:282]: Why do you need this?

**Review by moran711 — APPROVED** [review]

**Review by NikitaDenysenko — APPROVED** [review]

File: modules/order/order.resolver.js (hunk 1 above)
Comment (kapoliub) [reply modules/order/order.resolver.js]: fixed

File: resolvers.js (hunk 2 above)
Comment (kapoliub) [reply resolvers.js]: fixed

**Review by ivan-bonk — APPROVED** [review]

**Pull request merged**